src/Adapters/FHIRPatientAdapter.php

```diff
@@ -452,6 +453,22 @@ public function interfaceToModel( PatientInterface $patient )
         $fhirPatient->addTelecom( $email );
 
 
+        $address = new FHIRAddress();
+        $line = new FHIRString();
+        $line->setValue($patient->getStreet());
+        $address->addLine($line);
+        $city = new FHIRString();
+        $city->setValue($patient->getCity());
+        $address->setCity($city);
+
+        $district = new FHIRString();
+        $district->setValue($patient->getCountry());
```

Comment:
@Leo24 district should map to "county" not "country" also the lines (there could be multiple) of the address should be separated by "|" character.